contentcuration/contentcuration/static/js/edit_channel/file_upload/hbtemplates/format_multilanguage_dropdown.handlebars

```diff
@@ -0,0 +1,4 @@
+<option selected disabled><i>Select language</i></option>
```

Comment:
Created as a separate template so that we can render it as the languages get assigned/unassigned